OptionSelector/CQmodel.m

```diff
@@ -0,0 +1,30 @@
+//
+//  CQmodel.m
+//  OptionSelector
+//
+//  Created by Z on 8/14/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "CQmodel.h"
+
+@implementation CQModel
+
+-(void)setup{
```

Comment:
You could also make this a class method that returns the array, something like

    + (NSArray*) getModel {
      return @[fish, trees, squirrels];
    }

then used like

    [CQModel getModel];

Consider this kind of approach when you only ever need one instance of something. This way, you don't have to worry about accidentally creating/using different "versions" of them model (or about someone else misunderstanding your class and doing that).